examples/window-app/common/src/ZclCallbacks.cpp

```diff
@@ -39,83 +40,22 @@ using namespace ::chip::app::Clusters::WindowCovering;
 void MatterPostAttributeChangeCallback(const app::ConcreteAttributePath & attributePath, uint8_t mask, uint8_t type, uint16_t size,
                                        uint8_t * value)
 {
-    if (attributePath.mClusterId == app::Clusters::Identify::Id)
+    switch (attributePath.mClusterId)
     {
+    case Id:
+        MatterWindowCoveringClusterServerAttributeChangedCallback(attributePath);
```

Comment:
Again, the cluster should just register for these notifications directly, instead of relying on the app to notify it.